.zenodo.json

```
@@ -0,0 +1,60 @@
+{
+    "creators":[
+        {
+            "name": "SuperDARN Data Analysis Working Group"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Billett, D.D.",
+            "orcid": "0000-0002-8905-8609"
+        },
+        {
+            "affiliation": "The Unviersity Centre in Svalbard",
+            "name": "Bland, E.C.",
+            "orcid": "0000-0002-0252-0400"
+        },
+        {
+            "affiliation": "Naval Research Laboratory",
+            "name": "Burrell, A.G.",
+            "orcid": "0000-0001-8875-9326"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Kotyk, K."
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Ponomarenko, P.V.",
+            "orcid": "0000-0001-8407-0193"
+        },
+        {
+            "affiliation": "SRI International",
+            "name": "Reimer, A.S.",
+            "orcid": "0000-0002-4621-3453"
+        },
+        {
+            "affiliation": "University of Saskatchewan",
+            "name": "Schmidt, M. T."
```

```suggestion
            "name": "Schmidt, M.T."
```

removing space between initials for consistency